app/uk/gov/hmrc/exports/models/declaration/DUCR.scala

```diff
@@ -18,7 +18,9 @@ package uk.gov.hmrc.exports.models.declaration
 
 import play.api.libs.json.{Json, OFormat}
 
-case class DUCR(ducr: String)
+case class DUCR(ducr: String) {
+  def isDefined : Boolean = ! this.ducr.isEmpty
```

Comment:
I think if we called this `nonEmpty` it might be a little more intuitive

Also you can do the same logic with `this.ducr.nonEmpty` rather than having the inverse

Reply:
I was looking to avoid the negation... `nonEmpty` :man_facepalming: 
So close, yet so far away...

Reply:
This has been addressed.